### PR TITLE
Add repository topics to tag list

### PR DESCRIPTION
**Pull request by benfulton**
This will help users to find repositories in a specific subject that they are interested in, such as astronomy or physics

### CODE REVIEW

File: .github/scripts/generate-first-issues.py [hunk 1]
```diff
@@ -57,6 +58,17 @@ def generate_markdown(line):
...
+    url = api_topic_base.format(repo=repo)
+    response = requests.get(url, headers=headers, params=data)
+    if response.status_code != 200:
+        print(
+            "Issue with response %s: %s, sleeping"
+            % (response.status_code, response.reason)
+        )
+        time.sleep(60)
+        return None, None
+    topics = response.json()['names']
```
Comment (vsoch) [inline .github/scripts/generate-first-issues.py]: Is this guaranteed to be there? I think it's usually safest to do:

```python
response.json().get("names") or []
```

File: .github/scripts/generate-first-issues.py [hunk 2]
```diff
@@ -83,7 +95,7 @@ def generate_markdown(line):
         content = "---\n"
 
         # Add labels as tags
-        tags = list([x["name"] for x in issue["labels"]])
+        tags = list([x["name"] for x in issue["labels"]]) + topics
```
Comment (vsoch) [inline .github/scripts/generate-first-issues.py]: We likely want to ensure this set is unique - so do a list(set()) here

File: .github/scripts/generate-first-issues.py [hunk 3]
```diff
@@ -57,6 +58,17 @@ def generate_markdown(line):
 
     extra_tags = extra_tags.split(",")
     repo = "/".join(repo.split("/")[-2:])
+    url = api_topic_base.format(repo=repo)
```
Comment (vsoch) [inline .github/scripts/generate-first-issues.py]: And a nit - please write this into it's own modular function!

**Comment by vsoch** [comment]
If it's OK with you, I'm going to tweak this a bit locally and do a test run. Sound OK @benfulton ?

You likely after merge want to delete your fork  - we will be squashing and merging so the commits will no longer be in sync since you opened the PR from your main branch.

**Comment by benfulton** [comment]
Yep, go ahead!

**Comment by vsoch** [comment]
Thanks @benfulton!

**Comment by alansill** [comment]
Hmm, not sure if this is related, but the pull-down menus are not working on the [https://hpc.social/good-first-issues/](https://hpc.social/good-first-issues/) page for me ATM.

**Comment by alansill** [comment]
Deleted a comment about the pull-down menus. They work, but *only* if you click in the window for them at a place other than the "v" symbol!

**Comment by vsoch** [comment]
It's probably always been like that - front end development is not my forte! We can open an issue for it if you like @alansill

**Pull request merged**